pims/base_frames.py

```diff
@@ -33,6 +33,12 @@ def pixel_type(self):
         """Returns a numpy.dtype for the data type of the pixel values"""
         pass
 
+    @property
+    def dtype(self):
+        # The choice of using the separate name `pixel_type` was historical
+        # and needlessly made PIMS objects look less like numpy arrays.
+        return self.pixel_type
```

Comment:
Should this be passed through `numpy.dtype` for good measure. Have seen some instances where the type returned by `pixel_type` currently is not a `dtype`, but a `str` representing the `dtype`.